app/scripts/views/complete_sign_up.js

```diff
@@ -139,10 +141,22 @@ define(function (require, exports, module) {
             } else if (
                 AuthErrors.is(err, 'INVALID_VERIFICATION_CODE') ||
                 AuthErrors.is(err, 'INVALID_PARAMETER')) {
-              // These server says the verification code or any parameter is
-              // invalid. The entire link is damaged.
-              verificationInfo.markDamaged();
-              err = AuthErrors.toError('DAMAGED_VERIFICATION_LINK');
+
+              // When coming from sign-in confirmation verification, show a
+              // verification link expired error instead of damaged verification link.
+              // This error is generated because the link has already been used.
+              if(self.viewName === 'complete-signin'){
+
+                // Disable resending verification, can only be triggered from new sign-in
+                self._showResend = false;
+                verificationInfo.markExpired();
+                err = AuthErrors.toError('EXPIRED_VERIFICATION_LINK');
```

Comment:
@shane-tomlinson What are you thoughts on using a new error here `EXPIRED_SIGNIN_VERIFICATION_LINK`? Seems like if we keep the same one it might mess up our metrics.